mozilla_schema_generator/__main__.py

```diff
@@ -67,6 +74,79 @@ def generate_main_ping(config, out_dir, split, pretty):
     dump_schema(schemas, out_dir, pretty, version=4)
 
 
+@click.command()
+@click.argument(
+    'config-dir',
+    type=click.Path(
+        dir_okay=True,
+        file_okay=False,
+        writable=False,
+        exists=True,
+    ),
+    default=CONFIGS_DIR,
+)
+@click.option(
+    '--out-dir',
+    help=("The directory to write the schema files to. "
+          "If not provided, writes the schemas to stdout."),
+    type=click.Path(
+        dir_okay=True,
+        file_okay=False,
+        writable=True,
+    ),
+    required=False
+)
+@click.option(
+    '--split',
+    is_flag=True,
+    help=("If provided, splits the schema into "
+          "smaller sub-schemas"),
+)
+@click.option(
+    '--pretty',
+    is_flag=True,
+    help=("If specified, pretty-prints the JSON "
+          "schemas that are outputted. Otherwise "
+          "the schemas will be on one line."),
+)
+@click.option(
+    '--common-pings-config',
+    default="common_pings.json",
+    help=("File containing URLs to schemas and configs "
+          "of pings in the common ping format."),
+)
+def generate_common_pings(config_dir, out_dir, split, pretty, common_pings_config):
+    if out_dir:
+        out_dir = Path(out_dir)
+
+    common_pings = []
+
+    with open(common_pings_config, 'r') as f:
+        common_pings = json.load(f)
+
+    for common_ping in common_pings:
+        schema_generator = CommonPing(common_ping["schema_url"])
+
+        config_data = {}
+
+        if "config" in common_ping:
+            with open(config_dir / common_ping["config"], 'r') as f:
+                config_data = yaml.load(f)
+
+        m = re.match(SCHEMA_NAME_RE, common_ping["schema_url"])
+        name = m.group(1)
+        version = m.group(2)
+        config = Config(name, config_data)
+
+        if name == "main":
+            schemas = schema_generator.generate_schema(config, split=split)
+        else:
+            # do not split schemas except the main schema
```

Comment:
At this point we shouldn't ever be splitting the main ping either; we can simply remove that param (it's disabled below).